apps/jobs/views.py

```
@@ -13,32 +13,151 @@
+            404: openapi.Response(
+                description="데이터 없음",
+                examples={"application/json": {"message": "해당 ID의 기업을 찾을 수 없습니다."}}
+            )
+        }
+    )
+    def get(self, request, *args, **kwargs):
+        # 1. 부모 클래스의 retrieve 로직을 실행하여 데이터를 가져옵니다.
+        # 내부적으로 urls.py에서 넘겨받은 pk 값을 사용해 DB를 뒤집니다.
+        try:
+            return super().get(request, *args, **kwargs)
+        except:
```

Ruff (0.14.13)

Do not use bare `except`

(E722)